src/main/java/io/vertx/core/json/JsonArray.java

```diff
@@ -92,6 +92,21 @@ public JsonArray(Buffer buf) {
     }
   }
 
+  /**
+   * Create a JsonArray from a array of objects.
+   *
+   * @param args The objects into JsonArray.
+   */
+  public static JsonArray of(Object... args) {
+    JsonArray arr = new JsonArray();
+
+    for (Object o : args) {
+      arr.add(o);
+    }
+
+    return arr;
```

Comment:
How about creating a list of the right size from the beginning and then copying efficiently (system arraycopy internally) instead of iterating over the values?

```suggestion
    if (args == null) {
      throw new NullPointerException();
    }
    return new JsonArray(new ArrayList(Arrays.asList(args)));
```